core/lib/src/logger.rs

```diff
@@ -77,7 +82,7 @@ macro_rules! warn_ { ($($args:expr),+) => { log_!(warn: $($args),+); }; }
 impl log::Log for RocketLogger {
     #[inline(always)]
     fn enabled(&self, record: &log::Metadata) -> bool {
-        record.target().starts_with("launch") || record.level() <= self.0.max_log_level()
+        record.target().starts_with("launch") || self.0.max_log_level().is_some() && self.0.max_log_level().unwrap() >= record.level()
```

Comment:
Could this also be something like:
```rust
record.target().starts_with("launch") || if let Some(level) = self.0.max_log_level() { level >= record.level() } else { false }
```

It's probably better to avoid the unwrap to improve maintainability, but it does seem a shame to require the explicit `false`.

Reply:
I'm actually going to change this to

```
record.target().starts_with("launch") && self.0.max_log_level().is_some() || 
if let Some(level) = self.0.max_log_level() { 
   level >= record.level() 
} else { 
   false 
}
```

Because this also gets rid of the launch logs so we actually have no rocket logs whatsoever.